File: iPhone/VerBank-CSTSv3-IOSClientApi/CSTS3Comm/ipop/OP_Ctrl5101.m
//
//  OP_Ctrl5101.m
//  VerBank-IOSClientStation
//
//  Created by Allone on 15/10/21.
//  Copyright © 2015年 zhanglei. All rights reserved.
//

#import "OP_Ctrl5101.h"

static NSString *jsonId               = @"OP_Ctrl5101";

static NSString *fnStatus               = @"1";

@implementation OP_Ctrl5101

-(id)init {
    if (self = [super init]) {
        [super setEntry:@"jsonId" entry:jsonId];
    }
    return self;
}

jsonImplementionObject (FnStatus,          fnStatus)

@end

File: WeChatService/VerBank-Ctrl/src/allone/MTP/VerBank01/Ctrl/comm/ipop/OP_Ctrl5101.java
package allone.MTP.VerBank01.Ctrl.comm.ipop;

import allone.MTP.VerBank01.Ctrl.comm.structure.FnStatus;

/**
 * 
 * @Project: VerBank-Ctrl 
 * @Title: OP_Ctrl5101.java 
 * @Package allone.MTP.VerBank01.Ctrl.comm.ipop 
 * @Description: 行動GW-GetFnStatus2 
 * @author wyb wang.yubo@allone.cn 
 * @date 2015年10月15日 下午3:40:53 
 * @Copyright: 2015 www.allone.cn Inc. All rights reserved. 
 * @version V3.0.0   
 */
public class OP_Ctrl5101 extends CtrlServerOPFather {

	private static final long serialVersionUID = -8560898802046325126L;

	private FnStatus fnStatus;

	public OP_Ctrl5101(CtrlServerIPFather ip) {
		super(ip);
	}

	public FnStatus getFnStatus() {
		return fnStatus;
	}

	public void setFnStatus(FnStatus fnStatus) {
		this.fnStatus = fnStatus;
	}

}
